frontend/src/stories/Dropdown.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { Story, Meta } from "@storybook/react";
+import { useState } from "react";
+
+import Dropdown from "../app/commonComponents/Dropdown";
+
+const fruits = ["Apple", "Orange", "Pear"];
+
+// const fruitOptions = Object.entries(fruits).map(([value, key]) => ({
+//   value: value,
+//   label: key,
+// }));
```

Comment:
another way of doing this, is one format preferred?

Reply:
I have a slight preference for this format since it reduces duplication, but no strong opinions on it.

Reply:
Updated to that format in 9f88aa8.